app/retail/templates/leaderboard.html

```diff
@@ -14,46 +15,98 @@
     <div class="row no-gutter  text-uppercase">
         <div class="col-3" style="background: #eee; ">
             {% include 'shared/current_balance.html' %}
-            {% include 'shared/sidebar_search.html' %}
+            <!-- {% include 'shared/sidebar_search.html' %} -->
```

Comment:
i think this module should still be on the page to keep parity with the other pages, no?

Reply:
I dont know, it will show a search side bar, which might be misleading to the user.